packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1263,52 +1265,56 @@ module.exports = function (log, config, oauthdb) {
     });
   };
 
-  Mailer.prototype.postVerifyEmail = function (message) {
-    log.trace('mailer.postVerifyEmail', {
-      email: message.email,
-      uid: message.uid,
-    });
-
-    const templateName = 'postVerify';
+  ['postVerify', 'postVerifyNonMobile'].forEach((templateName) => {
     const subject = gettext(
       'Account verified. Next, sync another device to finish setup'
     );
-    const query = {};
-
     const action = gettext('Setup next device');
+    const isDesktopTemplate = templateName === 'postVerifyNonMobile';
+    const query = isDesktopTemplate ? { utm_source: 'fxa' } : {};
 
-    const links = this._generateLinks(
-      this.syncUrl,
-      message,
-      query,
-      templateName
-    );
+    Mailer.prototype[`${templateName}Email`] = async function (message) {
+      log.trace(`mailer.${templateName}Email`, {
+        email: message.email,
+        uid: message.uid,
+      });
 
-    const headers = {
-      'X-Link': links.link,
-    };
+      const links = this._generateLinks(
+        this.syncUrl,
+        message,
+        query,
+        templateName
+      );
 
-    return this.send({
-      ...message,
-      headers,
-      subject,
-      template: templateName,
-      templateValues: {
-        action,
-        androidLinkAttributes: linkAttributes(links.androidLink),
-        androidUrl: links.androidLink,
-        cadLinkAttributes: linkAttributes(links.link),
-        iosLinkAttributes: linkAttributes(links.iosLink),
-        iosUrl: links.iosLink,
-        link: links.link,
-        privacyUrl: links.privacyUrl,
-        style: message.style,
+      const headers = {
+        'X-Link': links.link,
+      };
+
+      return this.send({
+        ...message,
+        headers,
         subject,
-        supportLinkAttributes: links.supportLinkAttributes,
-        supportUrl: links.supportUrl,
-      },
-    });
-  };
+        template: templateName,
+        templateValues: {
+          action,
+          androidLinkAttributes: linkAttributes(links.androidLink),
+          androidUrl: links.androidLink,
+          isDesktopTemplate,
+          cadLinkAttributes: isDesktopTemplate
+            ? linkAttributes('https://www.firefox.com')
```

Comment:
@vbudhram should this use `config.smtp.redirectDomain`? It seems to use `localhost` when running `write-emails` locally, though I mostly didn't use it because we're only using it to actually _redirect_ the user, yet it is a link here. Would it be proper to use in this instance?

Reply:
We should probably expose new a config url to download Firefox browser, maybe something like `firefoxDesktopUrl` [here](https://github.com/mozilla/fxa/blob/2d566e40d74caadefaaf883fd3b28334eecde670/packages/fxa-auth-server/config/index.ts#L310) and corresponding `desktopLinkAttributes`. I think this should avoid issue above.